src/Application/Actions/Hooks/StripePaymentsUpdate.php

```diff
@@ -495,7 +495,7 @@ private function handleCashBalanceUpdate(Event $event, Response $response): Resp
             return $this->respond($response, new ActionPayload(200));
         }
 
-        $this->donationFundsNotifier->notifyRecieptOfAccountFunds($donorAccount, $transferAmount, $endingBalance);
+        $this->donationFundsNotifier->notifyRecieptOfAccountFunds($donorAccount, $transferAmount);
 
```

Comment:
no longer passing the endingBalance because the notifier will wait 30 seconds and pull a new balance from Stripe.